src/Package/Impl/Resources.resx

```diff
@@ -764,7 +764,10 @@ Additional information:
   <data name="HelpSearchWatermark" xml:space="preserve">
     <value>Type search term</value>
   </data>
-  <data name="VariableExplorer_ErrorEnvironment" xml:space="preserve">
-    <value>Error retrieving environment list</value>
+  <data name="Settings_EvaluateActiveBindings" xml:space="preserve">
+    <value>Evalaute active bindings.</value>
+  </data>
+  <data name="Settings_EvaluateActiveBindings_Description" xml:space="preserve">
+    <value>Show result of active binding evaluation in Variable Explorer and debugger tool windows. </value>
```

Comment:
This should make it very clear that this involves running code, potentially with side effects. Something like "Automatically evaluate active bindings and display the resulting value. Warning: when this option is on, inspecting active bindings in Variable Explorer and debugger tool windows may change program state."